### PR TITLE
(in) definite article + commas in basics/alias-string

### PR DESCRIPTION
**Pull request by wilzbach**
Found this during the review of #85. 

I am not 100% sure about the commas before "but"

### CODE REVIEW

File: basics/alias-strings.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ points total. Finally, `byGrapheme` performs rather expensive calculations
 to recognize that these two code points combine into single displayed
 character.
 
-Correct processing of Unicode can be very complicated but for most time D
+Correct processing of Unicode can be very complicated, but for most time D
```
Comment (ZombineDev) [inline basics/alias-strings.md]: "...but for most **of the** time**,** D developers..."

File: basics/alias-strings.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ This small example illustrates the difference in interpretation:
     import std.uni : byGrapheme;
     writeln(s.byGrapheme.walkLength); // 1
 
-Here actual array length of `s` is 3 because it contains 3 code units -
+Here the actual array length of `s` is 3 because it contains 3 code units -
```
Comment (JackStouffer) [inline basics/alias-strings.md]: Use `:` instead of `-`

**Comment by JackStouffer** [comment]
Fix nits then I'll merge

File: basics/alias-strings.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ points total. Finally, `byGrapheme` performs rather expensive calculations
 to recognize that these two code points combine into single displayed
 character.
 
-Correct processing of Unicode can be very complicated but for most time D
+Correct processing of Unicode can be very complicated, but for most of the time D
 developers can simply consider `string` variables as magical byte arrays and
```
Comment (wilzbach) [inline basics/alias-strings.md]: @ZombineDev you suggested to use a comma after "D developers" as I am not sure about this, I left the document in its original state.

File: basics/alias-strings.md [hunk 4]
```diff
@@ -27,9 +27,9 @@ the `to` method from `std.conv`:
     dstring myDstring = to!dstring(myString);
     string myString   = to!string(myDstring);
 
-This means that plain `string` is defined as an array of 8-bit Unicode [code
+This means that a plain `string` is defined as an array of 8-bit Unicode [code
 units](http://unicode.org/glossary/#code_unit). All array operations can be
-used with it but they will work on code unit level, not on character level. At
+used with it, but they will work on code unit level, not on character level. At
```
Comment (JackStouffer) [inline basics/alias-strings.md]: Would be clearer as

> used on strings, but

File: basics/alias-strings.md (hunk 3 above)
Comment (ZombineDev) [inline basics/alias-strings.md]: TBH, I'm not 100% sure either

Comment (ZombineDev) [inline basics/alias-strings.md]: @JackStouffer ?

Comment (JackStouffer) [inline basics/alias-strings.md]: It's right as is

Comment (ZombineDev) [inline basics/alias-strings.md]: Thanks for the clarification!

**Pull request merged**